src/js/client.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { Router, Route, IndexRoute, browserHistory } from 'react-router';
+import '../../public/css/main.scss';
+import AppRoutes from './routes/AppRoutes.jsx';
+
+
+// import App from './components/App.jsx';
+// import Sources from './components/Sources.jsx';
+// import Articles from './components/Articles.jsx';
```

Comment:
Commented out code again. This happens in several places some of which I will not comment on